lms/static/scripts/frontend_apps/components/SubmitGradeForm.js

```diff
@@ -20,18 +20,46 @@ import ValidationMessage from './ValidationMessage';
 export default function SubmitGradeForm({ disabled = false, student }) {
   const [showError, setShowError] = useState(false); // Is there a validation error message to show?
   const [gradeErrorMessage, setGradeErrorMessage] = useState(''); // The actual validation error message.
+  const [grade, setGrade] = useState('');
 
-  const [networkError, setNetworkError] = useState(''); // if there is an error when submitting grade
-  const [requestStatus, setRequestStatus] = useState(''); // ajax request state, one of ('', 'fetching', 'error', 'done')
+  const [networkError, setNetworkError] = useState(''); // If there is an error when submitting grade
+  const [requestStatus, setRequestStatus] = useState(''); // Ajax request state, one of ('', 'fetching', 'error', 'done')
```

Comment:
This and the following state have the same comment, so it's hard for me to understand what they mean. `requestStatus` sounds generic, but `fetchStatus` sounds specific to the fetch operation. Is `requestStatus` akin to `postStatus`?

Reply:
Comments are wrong indeed. I will also see if i can reuse some logic here so I don't have to replicate state for network request, it looks messy